client/my-sites/plugins/style.scss

```
@@ -110,3 +116,183 @@ body.is-section-plugins:not( .is-nav-unification ) {
 		padding: 0;
 	}
 }
+
+.plugin-details__layout {
+	@include grid-row( 4, 1 );
+	@include breakpoint-deprecated( '>1040px' ) {
+		@include display-grid;
+		@include grid-template-columns( 12, 24px, 1fr );
+		grid-gap: 24px;
+	}
+
+	.plugin-details__layout-col-left {
+		@include breakpoint-deprecated( '>1040px' ) {
+			@include grid-column( 1, 8 );
+		}
+	}
+
+	.plugin-details__layout-col-right {
+		@include breakpoint-deprecated( '>1040px' ) {
+			@include grid-column( 9, 4 );
+		}
+	}
+}
+
+.plugin-details__top-section {
+	border-bottom: 1px solid var( --studio-gray-5 );
+
+	.plugin-details__layout-col-left.no-cta {
+		@include grid-column( 1, 12 );
+	}
+
+	.plugin-details__layout-col-right.no-cta {
+		display: none;
+	}
+}
+
+.plugin-details__header {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	height: 400px;
+
+	.plugin-details__name, .plugin-details__price {
+		font-family: Recoleta;
```

Let's use `$brand-serif` instead of `Recoleta`
```suggestion
		font-family: $brand-serif;
```